xclim/sdba/adjustment.py

```diff
@@ -506,3 +510,133 @@ def _train(self, ref, hist):
     def _adjust(self, sim, interp="nearest"):
         factor = broadcast(self.ds.af, sim, group=self.group, interp=interp)
         return apply_correction(sim, factor, self.kind)
+
+
+class PrincipalComponent(BaseAdjustment):
+    r"""Principal Component adjustment.
```

Comment:
According to PEP 257, 

> The class constructor should be documented in the docstring for its `__init__` method. Individual methods should be documented by their own docstring.

Reply:
Well yeah ok. This was done because of the structure of the Adjustment objects with this `train / _train` scheme. But I have an idea.